test/distributed/fsdp/test_fsdp_mixed_precision.py

```diff
@@ -388,6 +402,100 @@ def test_mixed_precision_embedding_table(self):
             model.module.run_backward(loss)
             optim.step()
 
+    @skip_if_lt_x_gpu(2)
+    @skipIfNoTorchVision
+    def test_mixed_precision_resnet(self):
```

Comment:
Something I have been trying to do is write a short description of what the test does in the docstring to help readers know what to expect going in. Not sure if you think this is valuable in general or unnecessary.